pkg/agent/cniserver/pod_configuration.go

```diff
@@ -198,6 +199,10 @@ func (pc *podConfigurator) configureInterfaces(
 	hostIface := result.Interfaces[0]
 	containerIface := result.Interfaces[1]
 
+	if !createOVSPort {
+		return nil
+	}
+
 	// Delete veth pair if any failure occurs in later manipulation.
 	success := false
 	defer func() {
```

Comment:
I think you should still update the comments in line 214 - 216 (no more about workload container)?

BTW, probably change "Kubelet" to "kubelet".

Reply:
Update the comment. Done.